spec/models/manageiq/providers/redhat/infra_manager/ovirt_services/v4_spec.rb

```diff
@@ -328,7 +328,7 @@
 
       subject(:reconfigure_vm) { ems.vm_reconfigure(vm, :spec => spec) }
 
-      it 'calls the add command with mandatory params' do
+      it 'calls the remove command with mandatory params' do
```

Comment:
Ha whoops :)

Reply:
Yes, It was my previous mistake... sorry :-(